src/Common/Log.cs

```diff
@@ -54,5 +54,27 @@ public static void Write(string message, ConsoleColor color = ConsoleColor.Gray)
                 }
             }
         }
+
+        public static string ErrorLogFilePath
+        {
+            get { return errorLogFilePath; }
+            set
+            {
+                if (!Path.IsPathRooted(value))
+                    throw new ArgumentException($"Path '{value}' is not rooted.", nameof(value));
+                errorLogFilePath = value;

```

Comment:
This is a nit, but we like block statements under if (surround throw statement in curlies)